skfda/representation/_functional_data.py

```diff
@@ -819,6 +820,51 @@ def sum(  # noqa: WPS125
 
         return self
 
+    @overload
+    def cov(
+        self: T,
+        s_points: NDArrayFloat,
```

Comment:
As the minimum supported Python version is 3.8, I would set these parameters as positional-only.